test/objective.jl

```diff
@@ -65,23 +65,22 @@ function objectives_test(ModelType::Type{<:JuMP.AbstractModel}, VariableRefType:
     end
 
     @testset "Linear objective changes" begin
-        m = ModelType()
+        # Only works with JuMP model, not extensions.
```

Comment:
These testsets should be outside `objectives_test` and inside the `"Objectives for JuMP.Model"` testset. Currently the tests are run twice for no reason.

Reply:
Did you prefer this version? https://github.com/JuliaOpt/JuMP.jl/pull/2008/commits/a3694f93e1ad3138226366d3c66ca2197dbaa24a

Reply:
Yes

Reply:
Done in the last commit!